Formula/spdlog.rb

```diff
@@ -59,7 +60,8 @@ def install
       }
     EOS
 
-    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}", "-L#{Formula["fmt"].opt_lib}", "-lfmt", "-o", "test"
+    pkg_config_flags = `pkg-config --cflags --libs spdlog`.chomp.split
+    system ENV.cxx, "-std=c++11", "test.cpp", *pkg_config_flags, "-o", "test"
```

Comment:
As I mentioned in #79629, this breaks users who don't use `pkg-config`, so I don't think this is the right change to make.

Reply:
Alright. If you want, I can instead change tweakme.h to guard the `#define` in an ifndef, like:

```
#ifdnef SPDLOG_EXTERNAL_FMT
#define SPDLOG_EXTERNAL_FMT
#endif
```

Would you be happy with that?

Reply:
Alright. I'll push that then, and then we can hear what upstream says.